### DISCUSSION TITLE
tailwind lsp stops working in neovim without tailwind.config.js file in root dir

### DISCUSSION THREAD

**Discussion by AndreiMotinga** [rails/tailwindcss-rails#414] [Q&A]
I'm moving project from esbuild to importmaps and so removed node_modules and such. 

I noticed that tailwind lsp stoped working. 

with esbuild I had 2 things: 
1.   "tailwindcss": "^3.4.1", in package json and 
2.   tailwind.config.js. in root dir

it looks like if I install tailwindcss globally and create tailwind.config.js in root - I get the functionality back. 

so I wondered if there is a way to keep tailwind.config.js in root directory? 

or maybe lsp doesnt need that file?  

does anybody else have this issue?

**Comment by gczh** [comment]
Facing a similar issue here. Did the same step as you – removed node_modules after moving to importmaps

![CleanShot 2024-10-17 at 01 16 28@2x](https://github.com/user-attachments/assets/39a98abd-9e19-4f40-b5c4-4fafb48c2bfc)

**Reply by willian** [reply]
Have you or @AndreiMotinga found the solution?
